frameworks/PHP/symfony/app/Resources/views/base.html.twig

```diff
@@ -1,3 +1,4 @@
+{% spaceless %}
```

Comment:
spaceless should be removed if possible: it adds runtime overhead

Reply:
+1 we don't recommend doing it in real projects.

Reply:
I though it was done only when compiling the template to PHP. Thanks for the tip.

Reply:
The issue here is that without ``spaceless`` the resulting page is a bit bigger (41,27ko rather than 40,28 for fortune test). So in the benchmark it can impact the response transmission time. See https://www.techempower.com/benchmarks/#section=code&hw=ph&test=fortune

Reply:
Then, remove whitespace "by hand".